client/my-sites/domains/domain-management/transfer/transfer-out/warning.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { translate } from 'i18n-calypso';
+import {
+	domainManagementEditContactInfo,
+	domainManagementNameServers,
+} from 'calypso/my-sites/domains/paths';
+
+export default function TransferOutWarning( { domainName, selectedSiteSlug } ) {
+	return (
+		<p>
+			{ translate(
+				'Most transfers can be completed in a week or less; however depending on your domain it may take longer. Your domain will continue to work, but {{strong}}domain settings such as {{nameserversLink}}name servers{{/nameserversLink}} and {{contactInformationEditLink}}contact information{{/contactInformationEditLink}} cannot be changed during the transfer.{{/strong}} If you need to make changes, please do so before starting the transfer process.',
```

Comment:
I'm certainly splitting hairs but I believe the closing `{{/strong}}` should be located before the `.` at the end of the sentence.

Reply:
Yeah, it felt wrong to me too - thanks for confirming my fears 😬 I've moved the dot outside `strong` 👍